lab-maddy/src/component/category-form/index.js

```diff
@@ -0,0 +1,56 @@
+// ###### CategoryForm Component
```

Comment:
make sure to delete these comments / instructions from your code before submitting